spec/requests/workflows/workflows_skip_all_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Skip all in a workflow' do
+  let(:wf) { FactoryBot.create(:workflow_step) }
+  let(:druid) { wf.druid }
+  let(:workflow_template) { WorkflowTemplateLoader.load_as_xml('accessionWF') }
+
+  before do
+    allow(QueueService).to receive(:enqueue)
+  end
+
+  describe 'POST skip_all' do
+    let(:druid) { 'druid:bb123bb1234' }
+    let(:workflow) { 'accessionWF' }
+
+    let(:headers) { { 'Content-Type' => 'application/xml' } }
+    let(:xml_content) { '<?xml version="1.0"?><process name="skip-all" note="note"/>' }
+
+    context 'when the version is passed without context' do
+      it 'creates new workflows and skips all steps' do
+        expect do
+          post "/objects/#{druid}/workflows/#{workflow}?version=1"
+        end.to change(WorkflowStep, :count).by(10)
+        post("/objects/#{druid}/workflows/#{workflow}/skip-all", params: xml_content, headers:)
+        expect(WorkflowStep.last(10).map(&:status).uniq).to eq(['skipped'])
```

Comment:
will these be the only 10 steps in the DB at the time this test is running?  if so, you might be able to drop the `last(10)` to make that clearer?  otherwise, it might be good to add an `order` clause, since return order isn't guaranteed in the absence of one, and since relying on the fact that rows are usually returned in order of last update can sometimes lead to intermittently flappy tests).